.github/BOTMETA.yml

```
@@ -1434,6 +1434,10 @@ files:
     ignore: matze
     labels: zypper
     maintainers: $team_suse
+  $modules/zypper_repository_info.py:
+    ignore: tobiaszeuch181
+    labels: zypper
+    maintainers: $team_suse
```

You should not ignore yourself, but add yourself as a maintainer:
```suggestion
    labels: zypper
    maintainers: $team_suse tobiaszeuch181
```